javascript-and-json/main.js

```diff
@@ -0,0 +1,15 @@
+const library = [
+  { isbn: '1234567', title: 'It', author: 'Stephen King' },
+  { isbn: '789234', title: 'East of Eden', author: 'John Steinbeck' },
+  { isbn: '104324', title: 'The Great Gatsby', author: 'F. Scott Fitzgerald' },
+];
+
```

Comment:
Missing 2 `console.log`s here. 1 for just `books`, and the other for `typeof books`